### ISSUE TITLE
Invalid Bye assignment

### ISSUE THREAD

**Issue by gereons** [morphinepl/ANRTournament#22]
During the Iserlohn Regionals there was an invalid assignment of the Bye - it was not assigned to the lowest ranking player, but instead to the player in the second-to-last place. The tournament had 17 players, at the end of round 3 there were 3 players with 4 points and one with 2 points. While paring round 4, instead of giving the bye to the player with 2 points (which he obviously didn't have before), the bye was assigned to one of the 4-point players.

**Comment by tantauralus** [comment]
I'm sure you did check this: could it be, that if the player with the 2 points had recieved the bye a pairing from a previous round would have been the result and thus one from the 4 point players had to recieve the bye because two players may not play each other twice during swiss? And how many of the 4 point players have had a bye in one of the first 3 rounds? It might also be usefull to have the results and pairings of the first three rounds to help reproduce the potential error.

**Comment by gereons** [comment]
Yes, we did check, and it was easy to manually fix the pairings by assigning the Bye to the 2-point player, pair two of the 4-pointers and the remaining 4-pointer to a player with 6 points. As far as I remember, only one of the 4-point players had a bye before. 
I'm not sure if the TO did save the data from when this happened, I just helped with the manual repairing.
